torchaudio/functional.py

```diff
@@ -860,6 +860,172 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
     return output
 
 
+def gain(waveform, gain_db=1.0):
+    # type: (Tensor, float) -> Tensor
+    r"""Apply amplification or attenuation to the whole waveform.
+
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time).
+       gain_db (float) Gain adjustment in decibels (dB) (Default: `1.0`).
+
+    Returns:
+       torch.Tensor: the whole waveform amplified by gain_db.
+    """
+    if (gain_db == 0):
+        return waveform
+
+    ratio = 10 ** (gain_db / 20)
+
+    return waveform * ratio
+
+
+def scale_to_interval(waveform, interval=1.0):
+    # type: (Tensor, float) -> Tensor
+    r"""Scales the whole waveform to an interval.
```

Comment:
nit: "Scale the waveform to the interval [-interval_max, interval_max] across all dimensions."

nit: How about `interval_max` instead of `interval`?